examples/llm_inference/android/app/src/main/java/com/google/mediapipe/examples/llminference/Model.kt

```diff
@@ -11,52 +11,140 @@ enum class Model(
         topK = 64,
         topP = 0.95f
     ),
-    GEMMA3_GPU(
-        path = "/data/local/tmp/gemma3-1b-it-int4.task",
-        url = "https://huggingface.co/litert-community/Gemma3-1B-IT/resolve/main/gemma3-1b-it-int4.task",
+    GEMMA_3_1B_IT_GPU(
+        path = "/data/local/tmp/gemma-3-1b-it_multi-prefill-seq_q8_ekv2048.task",
+        url = "https://huggingface.co/litert-community/gemma-3-1b-it/resolve/main/gemma-3-1b-it_multi-prefill-seq_q8_ekv2048.task",
         licenseUrl = "https://huggingface.co/litert-community/Gemma3-1B-IT",
         needsAuth = true,
         preferredBackend = Backend.GPU,
-        uiState = GenericUiState(),
-        temperature = 1f,
+        thinking = false,
+        temperature = 1.0f,
         topK = 64,
         topP = 0.95f
     ),
-    DEEPSEEK_CPU(
-        path = "/data/local/tmp/llm/deepseek3k_q8_ekv1280.task",
-        url = "https://huggingface.co/litert-community/DeepSeek-R1-Distill-Qwen-1.5B/resolve/main/deepseek_q8_ekv1280.task",
+    GEMMA_2_2B_IT_CPU(
+        path = "/data/local/tmp/gemma-2-2b-it_multi-prefill-seq_q8_ekv1280.task",
+        url = "https://huggingface.co/litert-community/gemma-2-2b-it/resolve/main/gemma-2-2b-it_multi-prefill-seq_q8_ekv1280.task",
+        licenseUrl = "https://huggingface.co/litert-community/Gemma3-1B-IT",
+        needsAuth = true,
+        preferredBackend = Backend.CPU,
+        thinking = false,
+        temperature = 0.6f,
+        topK = 50,
+        topP = 0.9f
+    ),
+    DEEPSEEK_R1_DISTILL_QWEN_1_5_B(
+        path = "/data/local/tmp/DeepSeek-R1-Distill-Qwen-1.5B_multi-prefill-seq_q8_ekv1280.task",
+        url = "https://huggingface.co/litert-community/DeepSeek-R1-Distill-Qwen-1.5B/resolve/main//data/local/tmp/DeepSeek-R1-Distill-Qwen-1.5B_multi-prefill-seq_q8_ekv1280.task",
         licenseUrl = "",
         needsAuth = false,
-        preferredBackend = null,
-        uiState = DeepSeekUiState(),
+        preferredBackend = Backend.CPU,
+        thinking = true,
         temperature = 0.6f,
         topK = 40,
         topP = 0.7f
     ),
-    PHI4_CPU(
-        path = "/data/local/tmp/llm/phi4_q8_ekv1280.task",
-        url = "https://huggingface.co/litert-community/Phi-4-mini-instruct/resolve/main/phi4_q8_ekv1280.task",
+    LLAMA_3_2_1B_INSTRUCT(
+        path = "/data/local/tmp/Llama-3.2-1B-Instruct_multi-prefill-seq_q8_ekv1280.task",
+        url = "https://huggingface.co/litert-community/DeepSeek-R1-Distill-Qwen-1.5B/resolve/main/deepseek_q8_ekv1280.task",
+        licenseUrl = "",
+        needsAuth = false,
+        preferredBackend = Backend.CPU,
+        thinking = false,
+        temperature = 0.6f,
+        topK = 64,
+        topP = 0.9f
+    ),
+    LLAMA_3_2_3B_INSTRUCT(
+        path = "/data/local/tmp/Llama-3.2-3B-Instruct_multi-prefill-seq_q8_ekv1280.task",
+        url = "https://huggingface.co/litert-community/Llama-3.2-3B-Instruct/resolve/main/Llama-3.2-3B-Instruct_multi-prefill-seq_q8_ekv1280.task",
```

Comment:
I don't see this file in the community yet (same for some of the tasks file in this PR). Are they going to be available soon?

Reply:
Yes, by this afternoon